firefox-ios/firefox-ios-tests/Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -1099,7 +1094,8 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         let table = app.tables.element(boundBy: 0)
         
         screenState.tap(table.cells[AccessibilityIdentifiers.Settings.Browsing.autoPlay], to: AutoplaySettings)
-        
+        screenState.tap(table.cells["OpenWith.Setting"], to: MailAppSettings)
+
```

Comment:
Let me add `MailAppSettings` to this new `BrowsingSettings` state.
https://github.com/mozilla-mobile/firefox-ios/commit/059169a21c3fe1ef4b42fdbb0b847f68a80105f2#diff-2532808b9c654115bba24acf31ff36b0b52a04456a7e42b96018ff11554d0fc7